server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -155,7 +155,8 @@ public enum MetadataVersion {
     // Support for metadata.version feature flag and Removes min_version_level from the finalized version range that is written to ZooKeeper (KIP-778)
     IBP_3_3_IV0(5, "3.3", "IV0", false),
 
-    // Support NoopRecord for the cluster metadata log (KIP-835)
+    // In KRaft mode, use BrokerRegistrationChangeRecord instead of UnfenceBrokerRecord and FenceBrokerRecord.
+    // Also, use NoopRecord as specified in KIP-835.
```

Comment:
In general, do we want to decouple metadata versions from Kafka releases? For example, should we add this version instead of reusing `IBP_3_3_IV1`?
```
     IBP_3_3_IV2(7, "3.3", "IV2", true);
```
My understanding is that for IBP, Kafka added IVX so that users can deploy clusters from trunk and perform upgrade on clusters from trunk. cc @mumrah

Reply:
ok, I will create a new MetadataVersion